extensions/ql-vscode/src/view/model-editor/__tests__/ModelTypeDropdown.spec.tsx

```diff
@@ -34,7 +36,7 @@ describe(ModelTypeDropdown.name, () => {
     );
   });
 
-  it("allows changing the type to 'Type' for Ruby", async () => {
+  it("allows changing the type to 'Type' for Ruby in canary mode", async () => {
```

Comment:
This needs changing now that we've stopped using canary checks. Same with the following test.